base/cvd/cuttlefish/host/commands/cvd/instances/reset_client_utils.cpp

```diff
@@ -39,14 +39,10 @@
 namespace cuttlefish {
 
 Result<RunCvdProcessManager> RunCvdProcessManager::Get() {
-  RunCvdProcessCollector run_cvd_collector =
-      CF_EXPECT(RunCvdProcessCollector::Get());
-  RunCvdProcessManager run_cvd_processes_manager(std::move(run_cvd_collector));
-  return run_cvd_processes_manager;
+  return RunCvdProcessManager();
 }
 
-RunCvdProcessManager::RunCvdProcessManager(RunCvdProcessCollector&& collector)
-    : run_cvd_process_collector_(std::move(collector)) {}
+RunCvdProcessManager::RunCvdProcessManager() {}
```

Comment:
What's the point of having these two? `Get` doesn't seem like it could fail now.